### PR TITLE
Remove GPL from our README.

### PR DESCRIPTION
**Pull request by arokem**
Also, added a few more small things + links.

Follow up to #1285

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -39,13 +39,20 @@ You can find our sources and single-click downloads:
 .. _Documentation: http://dipy.org
 .. _current trunk: http://github.com/nipy/dipy/archives/master
 
+
+Installing DIPY
+===============
+
+For installation instructions, please read our `documentation <http://nipy.org/dipy/installation.html>`_.
```
Comment (Garyfallidis) [inline README.rst]: I think we should start with
conda install -c conda-forge dipy vtk
and then provide the link to the rest of the installation information.

File: README.rst [hunk 2]
```diff
@@ -39,13 +39,20 @@ You can find our sources and single-click downloads:
...
 =======
 
 DIPY is licensed under the terms of the BSD license.
-Please see the LICENSE file in the dipy distribution.
+Please see `LICENSE file <https://github.com/nipy/dipy/blob/master/LICENSE>`_.
+
+Contributing
+============
 
-DIPY uses other libraries also licensed under the BSD or the
```
Comment (Garyfallidis) [inline README.rst]: Feels great! :)

**Comment by arokem** [comment]
How about adding both pip and conda install instructions here, and then
refer them to the documentation for detailed instructions?

**Comment by Garyfallidis** [comment]
I would only keep conda in the Readme. Fast, easy and complete installation both api and viz. With pip things are a bit more complicated and there is not pip for vtk. Let's make our users happy. With one line you get it all :)

**Comment by matthew-brett** [comment]
That keeps your users happy only if they are already using conda.

**Comment by matthew-brett** [comment]
Ouch :)

**Comment by matthew-brett** [comment]
Yes, I do - but even if I didn't, I don't think I'd try and force someone who was thus far using pip to use conda.   There's a long discussion over on Scipy github as to why Anaconda monoculture might be undesirable.

**Comment by Garyfallidis** [comment]
Oops. Can send me the link to the discussion?

**Comment by matthew-brett** [comment]
Here's one of Gael's comments in the discussion : https://github.com/scipy/scipy.org/pull/179#issuecomment-247149925

**Comment by Garyfallidis** [comment]
Okay I see the issue @matthew-brett . The problem we have with pip is that there is no pip for vtk.

**Comment by Garyfallidis** [comment]
And I actually do not understand why there is no pip for vtk. It would be wonderful to have that.

**Comment by Garyfallidis** [comment]
Do you know why?

**Comment by matthew-brett** [comment]
Yes, I do - I'm on the VTK mailing list and I was on a group hangout with the VTK guys about this a few weeks ago.   The main problem is compiling a VTK wheel that can work with a reasonable number of backends.  They are working on it at the moment, I guess that it will be done in a few months.

**Comment by Garyfallidis** [comment]
Cool, that is great. So, what do you suggest put pip in the readme now or wait for the VTK in pypi so that the information is more complete?

**Comment by matthew-brett** [comment]
I suggest keeping pip in the readme with the caveat that the user should install VTK separately when using pip (via apt or homebrew).  I think it's OK to recommend conda over pip, I'm just saying that you should keep in mind that a lot of people do use pip, and some people have a strong preference for pip.

**Comment by Garyfallidis** [comment]
Understood. We can add pip @arokem. Thanks @matthew-brett for pointing this out. +1

**Comment by arokem** [comment]
All good. I just reverted that last change, so we now have pip and conda.

**Comment by arokem** [comment]
BTW -- still can't seem to install VTK on Python 3, even with conda. Added a comment about that.

**Comment by grlee77** [comment]
> BTW -- still can't seem to install VTK on Python 3, even with conda. Added a comment about that.

If you are on Windows or Linux, VTK 7.1.1 should be available for Python 2.7, 3.5 and 3.6 from conda-forge.  If you are not on OS X, can you specify the problem you encountered with conda?  We don't have OS X builds there due to build timeouts on the CI services.  

p.s.  You can see the specific package versions available at:  https://anaconda.org/conda-forge/vtk/files

The conda-forge build script is compatible with OS X if you want to use it to build your own local package.  I doubt you want to go into that level of detail in the `Dipy` docs, but if interested the procedure is:

```
1.) clone https://github.com/conda-forge/vtk-feedstock
2.) cd vtk-feedstock
3.) edit recipe/meta.yaml to remove the "skip" line so an OS X build won't be skipped.
4.) conda build -c conda-forge --python 3.6 recipe
5.) conda install --use-local vtk
```

**Comment by arokem** [comment]
Thanks! 

I see: 

```
chlosyne:~  $conda install -c conda-forge vtk
Fetching package metadata ...........
Solving package specifications: .

UnsatisfiableError: The following specifications were found to be in conflict:
  - python 3.5*
  - vtk -> python 2.7*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by arokem** [comment]
Should say: on OSX. I will try your more detailed instructions, but I definitely don't think that we want that level of detail in our README.

**Comment by arokem** [comment]
I can't get the more detailed instructions you provided to work for me. I get: 

```
chlosyne:vtk-feedstock (master) $conda build -c conda-forge --python 3.5 recipe
Traceback (most recent call last):
  File "/Users/arokem/anaconda3/bin/conda-build", line 4, in <module>
    import conda_build.cli.main_build
  File "/Users/arokem/anaconda3/lib/python3.5/site-packages/conda_build/cli/main_build.py", line 16, in <module>
    import conda_build.api as api
  File "/Users/arokem/anaconda3/lib/python3.5/site-packages/conda_build/api.py", line 22, in <module>
    from conda_build.config import Config, get_or_merge_config, DEFAULT_PREFIX_LENGTH as _prefix_length
  File "/Users/arokem/anaconda3/lib/python3.5/site-packages/conda_build/config.py", line 14, in <module>
    from .conda_interface import string_types, binstar_upload
  File "/Users/arokem/anaconda3/lib/python3.5/site-packages/conda_build/conda_interface.py", line 21, in <module>
    from conda.resolve import MatchSpec, NoPackagesFound, Resolve, Unsatisfiable, normalized_version  # NOQA
ImportError: cannot import name 'NoPackagesFound'
```

**Comment by skoudoro** [comment]
With OS X,  you have to use the [clinicalgraphics channel](https://anaconda.org/clinicalgraphics/vtk)

`conda install -c clinicalgraphics vtk`

**Comment by arokem** [comment]
Huh. Interesting. That doesn't work with Python 3.5 (which is what I am using atm), but does seem like it would work with 3.6. Thanks for pointing that out. Even so: I don't think we want this level of detail in the README.

**Comment by grlee77** [comment]
> I can't get the more detailed instructions you provided to work for me

I have not seen that error, but it could be related to the version of conda or conda-build used.  conda-forge currently uses conda 4.3.22 and conda-build 2.1.17.  You can install those versions via:

conda install -c conda-forge conda conda-build

(These conda tools must be installed into the base anaconda environment)

**Comment by skoudoro** [comment]
I agree, we do not need this level of detail in the README. 

vtk installation seems to be quite limited with conda on OSX. (only python 3.6, not even python 2.7)

File: README.rst [hunk 3]
```diff
@@ -39,13 +39,29 @@ You can find our sources and single-click downloads:
...
 
+
+Installing DIPY
+===============
+
+DIPY can be installed using `pip`:
+
+    pip install dipy
+
+or using `conda` (only on Python 2):
```
Comment (Garyfallidis) [inline README.rst]: Hello. As far as I know there are no issues with Python 3. Please remove.

Comment (arokem) [reply README.rst]: Did something change since our conversation two weeks ago?

Comment (grlee77) [reply README.rst]: @arokem : The issue was OS X, not Python 3.  The current availability of VTK 7.1.1 is Python 2.7, 3.5 and 3.6 for linux-64, win-32 and win-64.  There are no `conda-forge` VTK packages for OS X (or for 32-bit linux).

I did recently upload VTK 7.1.1 packages built for OS X [in my channel](https://anaconda.org/grlee77/vtk) if you want to try them out.  As of now these match the `conda-forge` recipe, but these have not undergone the automated CI testing process and I can't guarantee to always keep them in sync going forward, so I wouldn't mention them in the README.

**Comment by arokem** [comment]
OK - thanks for the explanation and for all your work on that!

File: README.rst [hunk 4]
```diff
@@ -39,13 +39,29 @@ You can find our sources and single-click downloads:
...
+Please see `LICENSE file <https://github.com/nipy/dipy/blob/master/LICENSE>`_.
+
+Contributing
+============
 
-DIPY uses other libraries also licensed under the BSD or the
-MIT licenses, with the only exception of the SHORE module which
-optionally uses the cvxopt library. Cvxopt is licensed
-under the GPL license.
+We welcome contributions from the community. Please read our `contributor guidlines <https://github.com/nipy/dipy/blob/master/CONTRIBUTING.md>`.
```
Comment (skoudoro) [inline README.rst]: `guidlines` to `guidelines`

Comment (skoudoro) [reply README.rst]: I think, it is ready to go after this correction. Any other suggestions ?

**Comment by Garyfallidis** [comment]
Thanks! Free again :)

**Comment by arokem** [comment]
https://www.youtube.com/watch?v=7TNecaCESbE

**Pull request merged**